### PR TITLE
feat(networkfirewall): add new check `networkfirewall_policy_default_action_fragmented_packets `

### PR DESCRIPTION
**Pull request by HugoPBrito**
### Context

When configuring firewall policies for AWS Network Firewall, handling fragmented packets correctly is crucial to maintaining network security. Fragmented packets can pose security risks as they may bypass certain packet filters or evade intrusion detection systems. Thus, the stateless action for such packets should default to either Drop or Forward, ensuring that no unintended traffic is allowed through.

### Description

This check ensures that the default stateless action for fragmented packets in an AWS Network Firewall policy is set to either Drop or Forward. Allowing fragmented packets to pass can lead to security vulnerabilities, as such traffic might bypass critical packet filtering and evasion mechanisms.

### Checklist

- Are there new checks included in this PR? Yes.
    - If so, do we need to update permissions for the provider? No.
- [x] Review if the code is being covered by tests.
- [x] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
- [x] Review if backport is needed.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by HugoPBrito** [comment]
I'm open to new CheckIDs, I wasn't able to figure out a shorter one.

**Comment by HugoPBrito** [comment]
I considered some additional possible check names:

1. `networkfirewall_policy_default_stateless_action_secure_frag_packets`
2. `networkfirewall_policy_secure_fragmented_packets`
3. `networkfirewall_policy_drop_or_forward_fragmented_packets`

File: prowler/providers/aws/services/networkfirewall/networkfirewall_default_stateless_action_drop_forward/networkfirewall_default_stateless_action_drop_forward.metadata.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "networkfirewall_default_stateless_action_drop_forward",
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_default_stateless_action_drop_forward/networkfirewall_default_stateless_action_drop_forward.metadata.json]:
```suggestion
  "CheckID": "networkfirewall_policy_default_action_fragmented_packets",
```

File: prowler/providers/aws/services/networkfirewall/networkfirewall_default_stateless_action_drop_forward/networkfirewall_default_stateless_action_drop_forward.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    def execute(self):
+        findings = []
+        for arn, firewall in networkfirewall_client.network_firewalls.items():
+            report = Check_Report_AWS(self.metadata())
+            report.region = firewall.region
+            report.resource_id = firewall.name
+            report.resource_arn = arn
+            report.resource_tags = firewall.tags
+            report.status = "FAIL"
+            report.status_extended = f"Network Firewall {firewall.name} default stateless action is not set to drop or forward."
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_default_stateless_action_drop_forward/networkfirewall_default_stateless_action_drop_forward.py]:
```suggestion
            report.status_extended = f"Network Firewall {firewall.name} policy does not drop or forward fragmented packets by default."
```

File: prowler/providers/aws/services/networkfirewall/networkfirewall_default_stateless_action_drop_forward/networkfirewall_default_stateless_action_drop_forward.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+            for action in firewall.default_stateless_frag_actions:
+                if action == "aws:drop" or action == "aws:forward_to_sfe":
+                    report.status = "PASS"
+                    report.status_extended = f"Network Firewall {firewall.name} default stateless action is set to drop or forward."
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_default_stateless_action_drop_forward/networkfirewall_default_stateless_action_drop_forward.py]:
```suggestion
                    report.status_extended = f"Network Firewall {firewall.name} policy does drop or forward fragmented packets by default."
```

File: prowler/providers/aws/services/networkfirewall/networkfirewall_service.py
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_service.py:1]: Where is the `default_stateless_frag_actions` attribute filled?

File: prowler/providers/aws/services/networkfirewall/networkfirewall_default_stateless_action_drop_forward/networkfirewall_default_stateless_action_drop_forward.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            report = Check_Report_AWS(self.metadata())
+            report.region = firewall.region
+            report.resource_id = firewall.name
+            report.resource_arn = arn
+            report.resource_tags = firewall.tags
+            report.status = "FAIL"
+            report.status_extended = f"Network Firewall {firewall.name} default stateless action is not set to drop or forward."
+
+            for action in firewall.default_stateless_frag_actions:
+                if action == "aws:drop" or action == "aws:forward_to_sfe":
```
Comment (MrCloudSec) [inline prowler/providers/aws/services/networkfirewall/networkfirewall_default_stateless_action_drop_forward/networkfirewall_default_stateless_action_drop_forward.py]:
```suggestion
                if "aws:drop" in firewall.default_stateless_frag_actions or "aws:forward_to_sfe" in firewall.default_stateless_frag_actions:
```

**Review by MrCloudSec — APPROVED** [review]
👏🏼 👏🏼

**Pull request merged**